tests/linearizability/failpoints.go

```diff
@@ -47,3 +53,51 @@ func (f killFailpoint) Trigger(ctx context.Context, clus *e2e.EtcdProcessCluster
 	}
 	return nil
 }
+
+type goFailpoint struct {
+	failpoint string
+	payload   string
+}
+
+func (f goFailpoint) Trigger(ctx context.Context, clus *e2e.EtcdProcessCluster) error {
```

Comment:
Please add method to support deleting the `failpoints`, so that test cases can reset the environment after finished.

Reply:
I don't understand what you mean by reseting the environment. Every e2e tests should spin their own etcd cluster so why I should delete the failpoint.

Reply:
You are right!